src/functions-and-arrays.js

```diff
@@ -1,19 +1,35 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 < num2) return num2
+   else 
+   return num1
+  }
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
-
+function findLongestWord(arr) {
+  let longestWord = arr[0]
+for (let words of arr) {
```

Comment:
Since you initialized `longestWord` with `arr[0]`, you can start the loop from the second array element. You would need a normal for loop in this case, where you start the index with 1.
` for (let i=1; i < arr.length; i++)`